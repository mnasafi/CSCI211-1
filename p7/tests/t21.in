echo t21: Insert days of week into tree, then call balanced
insert Sunday
insert Monday
insert Tuesday
insert Friday
insert Saturday
insert Thursday
insert Wednesday
breadth
balanced
