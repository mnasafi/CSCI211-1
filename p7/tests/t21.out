t21: Insert days of week into tree, then call balanced
{Sunday, Monday, Tuesday, Friday, Saturday, Thursday, Wednesday}
Tree is balanced.
